custom-generators/LatGenerator.js

```diff
@@ -0,0 +1,55 @@
+var chance = require("chance").Chance();
+
+function LatGenerator() {}
+
+LatGenerator.prototype.generate = function(system) {
+  var labels = system.labels;
+
+  for (var i = labels.length - 1; i >= 0; i--) {
+    var label = labels[i]
+
+    // Asia/Japan/Tokyo:
+    if(label == "Tokyo")
```

Comment:
Sorry @rjacolin I just saw this now but it is better here to use `switch/case` syntax...